sections/staking/components/StakingInfo/StakingInfo.tsx

```diff
@@ -139,44 +134,54 @@ const StakingInfo: React.FC<StakingInfoProps> = ({ isMint }) => {
 				: changedStakedValue.multipliedBy(SNXRate).dividedBy(mintAdditionalDebt).multipliedBy(100)
 			: toBigNumber(100).dividedBy(changedDebt.dividedBy(SNXRate).dividedBy(collateral));
 
-		return [
-			{
-				title: t('staking.info.table.not-staked'),
-				value: sanitiseValue(unstakedCollateral),
-				changedValue: sanitiseValue(changedNotStakedValue),
-				currencyKey: CryptoCurrency.SNX,
-			},
-			{
-				title: t('staking.info.table.staked'),
-				value: sanitiseValue(stakedCollateral),
-				changedValue: sanitiseValue(changedStakedValue),
-				currencyKey: CryptoCurrency.SNX,
-			},
-			{
-				title: t('staking.info.table.transferable'),
-				value: sanitiseValue(transferableCollateral),
-				changedValue: sanitiseValue(changedTransferable),
-				currencyKey: CryptoCurrency.SNX,
-			},
-			{
-				title: t('staking.info.table.locked'),
-				value: sanitiseValue(lockedCollateral),
-				changedValue: sanitiseValue(changedLocked),
-				currencyKey: CryptoCurrency.SNX,
-			},
-			{
-				title: t('staking.info.table.c-ratio'),
-				value: sanitiseValue(toBigNumber(100).dividedBy(currentCRatio)),
-				changedValue: sanitiseValue(changeCRatio),
-				currencyKey: '%',
-			},
-			{
-				title: t('staking.info.table.debt'),
-				value: sanitiseValue(debtBalance),
-				changedValue: sanitiseValue(changedDebt),
-				currencyKey: Synths.sUSD,
-			},
-		];
+		return {
+			barRows: [
+				{
+					title: t('staking.info.table.staked'),
+					value: sanitiseValue(stakedCollateral),
+					changedValue: sanitiseValue(changedStakedValue),
+					percentage: collateral.isZero()
+						? toBigNumber(0)
+						: sanitiseValue(stakedCollateral).dividedBy(collateral),
+					changedPercentage: collateral.isZero()
+						? toBigNumber(0)
+						: sanitiseValue(changedStakedValue).dividedBy(collateral),
+					currencyKey: CryptoCurrency.SNX,
+				},
+				{
+					title: t('staking.info.table.transferable'),
+					value: sanitiseValue(transferableCollateral),
+					changedValue: sanitiseValue(changedTransferable),
+					percentage: collateral.isZero()
+						? toBigNumber(0)
+						: sanitiseValue(transferableCollateral).dividedBy(sanitiseValue(collateral)),
+					changedPercentage: collateral.isZero()
+						? toBigNumber(0)
+						: sanitiseValue(changedTransferable).dividedBy(sanitiseValue(collateral)),
+					currencyKey: CryptoCurrency.SNX,
+				},
+			],
+			dataRows: [
+				{
+					title: t('staking.info.table.c-ratio'),
+					value: sanitiseValue(toBigNumber(100).dividedBy(currentCRatio)),
+					changedValue: sanitiseValue(changeCRatio),
+					currencyKey: '%',
+				},
+				{
+					title: t('staking.info.table.susd-balance'),
+					value: sanitiseValue(sUSDBalance),
+					changedValue: sanitiseValue(changedDebt),
```

Comment:
@clementbalestrat this changedValue is incorrect, its different to the debt row it must add the current sUSD balance of the user as well.

Scenario 1:

I have no debt because I have not staked, my current sUSD balance is 75.

If i mint max I will receive 100 sUSD in debt, therefore this row should show 75 + 100 = 175 after minting